client/src/components/Issue.js

```diff
@@ -40,127 +41,136 @@ const Issue = ({
-                    d="M1.5 8a6.5 6.5 0 0110.65-5.003.75.75 0 00.959-1.153 8 8 0 102.592 8.33.75.75 0 10-1.444-.407A6.5 6.5 0 011.5 8zM8 12a1 1 0 100-2 1 1 0 000 2zm0-8a.75.75 0 01.75.75v3.5a.75.75 0 11-1.5 0v-3.5A.75.75 0 018 4zm4.78 4.28l3-3a.75.75 0 00-1.06-1.06l-2.47 2.47-.97-.97a.749.749 0 10-1.06 1.06l1.5 1.5a.75.75 0 001.06 0z"
-                  ></path>
-                </svg>
-              )}
-              {draft !== undefined && state === 'open' && (
-                <svg
-                  height="16"
-                  className="octicon octicon-git-pull-request open flex-shrink-0"
-                  viewBox="0 0 16 16"
-                  version="1.1"
-                  width="16"
-                  aria-hidden="true"
-                >
-                  <path
-                    fillRule="evenodd"
-                    d="M7.177 3.073L9.573.677A.25.25 0 0110 .854v4.792a.25.25 0 01-.427.177L7.177 3.427a.25.25 0 010-.354zM3.75 2.5a.75.75 0 100 1.5.75.75 0 000-1.5zm-2.25.75a2.25 2.25 0 113 2.122v5.256a2.251 2.251 0 11-1.5 0V5.372A2.25 2.25 0 011.5 3.25zM11 2.5h-1V4h1a1 1 0 011 1v5.628a2.251 2.251 0 101.5 0V5A2.5 2.5 0 0011 2.5zm1 10.25a.75.75 0 111.5 0 .75.75 0 01-1.5 0zM3.75 12a.75.75 0 100 1.5.75.75 0 000-1.5z"
-                  ></path>
-                </svg>
-              )}
-              {draft !== undefined && state === 'closed' && (
-                <svg
-                  height="16"
-                  className="octicon octicon-git-pull-request closed flex-shrink-0"
-                  viewBox="0 0 16 16"
-                  version="1.1"
-                  width="16"
-                  aria-hidden="true"
-                >
-                  <path
-                    fillRule="evenodd"
-                    d="M7.177 3.073L9.573.677A.25.25 0 0110 .854v4.792a.25.25 0 01-.427.177L7.177 3.427a.25.25 0 010-.354zM3.75 2.5a.75.75 0 100 1.5.75.75 0 000-1.5zm-2.25.75a2.25 2.25 0 113 2.122v5.256a2.251 2.251 0 11-1.5 0V5.372A2.25 2.25 0 011.5 3.25zM11 2.5h-1V4h1a1 1 0 011 1v5.628a2.251 2.251 0 101.5 0V5A2.5 2.5 0 0011 2.5zm1 10.25a.75.75 0 111.5 0 .75.75 0 01-1.5 0zM3.75 12a.75.75 0 100 1.5.75.75 0 000-1.5z"
-                  ></path>
-                </svg>
-              )}{' '}
-              {title}
-            </a>
-          </p>
-
-          <div className="issue-details">
-            <div className="issue-repo">
-              <a
-                target="_blank"
-                rel="noopener noreferrer"
-                href={getRepoUrlFromIssueUrl(html_url)}
-              >
-                {getRepoNameFromIssueUrl(html_url)}
+  
+  // pddemo/demo creates a new fake issue every minute
+  // Avatar url 10575 is GitHub Learning Lab which creates issues as pieces of tutorials for users to follow as 
+  // they create projects. These are trash for our users to try and sort through, so they are excluded here
+  if (repoName !== "pddemo/demo" && avatar_url !== "https://avatars0.githubusercontent.com/in/10572?v=4") {    
```

Comment:
It would be nice if this exception wasn't hardcoded, or at least if there was a way to turn it on/off.